application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -169,22 +173,18 @@ private RestAction<ForumPost> createForumPost(ModalInteractionEvent event, User
     private RestAction<Message> dmUser(MessageChannelUnion sourceChannel, ForumPost forumPost,
             Guild guild) {
 
+        String messageTemplate =
+                """
+                        Hello %s 👋 You have asked a question in the wrong channel %s. Not a big deal, but none of the experts who could help you are reading your question there 🙁
```

Comment:
You get odd spaces for formatting either. Example:
```java
Hello[two spaces]👋 You have asked a question i nthe wrong channel[one space].
```
Instead, you have to write it as `Hello %s👋 You have asked a question in the wrong channel%s.`

and then format one as `"[one space]on " + ...` and the other as `" " + forumPost.author.getAsMention()`